### PR TITLE
fundingRate, previousFundingRate and nextFundingRate naming fix #11298

### LINKED ISSUE #11298: fundingRate, previousFundingRate and nextFundingRate
related: PR #10512 #10225
I think in the return data structure of fetchFundingRate api, should contain `fundingRate` and `nextFundingRate`, but not `previousFundingRate`.

`previousFundingRate` means the fundingRate that already been charged, this field should not be included in fetchFundingRate api.
`fundingRate` means the fundingRate that will be charged in the next 1-8hours(or according to the exchange settings). fundingRate should never be None.
`nextFundingRate` means the fundingRate that will be charged next time, after fundingRate being charged. Some exchanges don't have this field, like ftx.

In the current ccxt implementation, `previousFundingRate` is confused with `fundingRate`.  In the most exchanges' ccxt implementation, `previousFundingRate` should rename to `fundingRate`. In ftx, `nextFundingRate` should rename to `fundingRate`. In okex, `nextFundingRate` should rename to `fundingRate`, and in okex, `nextFundingRate`  itself should be reimplemented.

### PR DESCRIPTION
**Pull request by zsyh**
fix #11298

### CODE REVIEW

**Comment by frosty00** [comment]
I don't think we should remove previousFundingRate here

**Comment by kroitor** [comment]
@zsyh i would agree with @frosty00 , i think we should keep all three.

**Comment by zsyh** [comment]
then, for example huobi, which field is previousFundingRate?
huobi_funding_rate_fields

**Comment by kroitor** [comment]
> then, for example huobi, which field is previousFundingRate? Leaving it undefined? Or same to fundingRate? 

If the exchange does not provide one of the three fields – leave it `undefined`.

**Comment by kroitor** [comment]
@zsyh lmk if you're going to add those last edits, I will review and merge asap. Thanks!

**Comment by zsyh** [comment]
I'll add previousFundingRate.

File: js/bybit.js [hunk 1]
```diff
@@ -1237,9 +1237,9 @@ module.exports = class bybit extends Exchange {
         // }
         //
         const result = this.safeValue (response, 'result');
-        const nextFundingRate = this.safeNumber (result, 'funding_rate');
-        const previousFundingTime = this.safeInteger (result, 'funding_rate_timestamp') * 1000;
-        const nextFundingTime = previousFundingTime + (8 * 3600000);
+        const fundingRate = this.safeNumber (result, 'funding_rate');
+        const fundingTime = this.safeInteger (result, 'funding_rate_timestamp') * 1000;
+        const nextFundingTime = fundingTime + (8 * 3600000);
```
Comment (kroitor) [inline js/bybit.js:1242]: We can't use `+` here, it violates the following rule: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#adding-numbers-and-concatenating-strings

Should be:

```JavaScript
        const nextFundingTime = this.sum (fundingTime, 8 * 3600000);
```

**Pull request merged**